docs/UserGuide.md

```diff
@@ -217,20 +217,33 @@ Update details of an existing internship entry.
 **Examples**:
 * `modify 1 c/Jane Street ro/Coffee maker a/Yet to apply s/20/01/2023 de/29/11/2022 du/3 re/C++ re/Coffee`
 
-### Filter Internships by Name and Keyword: `filter`
+### Filter Internships by Category and Keyword/Range: `filter`
 
-Display internships matching specific category keywords.
+Want to focus only on relevant internship entries based on certain constraints? Filter internship entries by inputting any category. There are 2 category types: type 1 accepting text inputs and type 2 accepting a range of values. The category types are as follows:
 
-**Format**: `filter [category]/[keyword] ...`
+* Category type 1: company name, role, requirements
+* Category type 2: duration, deadline
+
```

Comment:
should use the github flavored markdowns (the one with the white background and red text font) to standardise ur style for CATEGORY_TYPE1/2 for this entire section